### PR TITLE
use separate VC processes for the local sim by default

### PR DESCRIPTION
**Pull request by onqtam**
Also fixed finalization - properly distributing the keys between BNs & VCs
And added a new `altona-vc` Makefile target that would use a VC for altona

### CODE REVIEW

File: tests/simulation/vars.sh [hunk 1]
```diff
@@ -24,6 +24,7 @@ TOTAL_NODES=${NODES:-4}
 TOTAL_USER_NODES=${USER_NODES:-0}
 TOTAL_SYSTEM_NODES=$(( TOTAL_NODES - TOTAL_USER_NODES ))
 BOOTSTRAP_NODE=$(( TOTAL_NODES - 1 ))
+USE_BN_VC_VALIDATOR_SPLIT=${BN_VC_VALIDATOR_SPLIT:-yes}
```
Comment (onqtam) [inline tests/simulation/vars.sh:27]: yes by default - seems stable enough

**Comment by onqtam** [comment]
2 of the builds fail because of testnet finalization... but it seems that there have been plenty of such failures in the past couple of days + the local sim passes locally.

**Comment by tersec** [comment]
> 2 of the builds fail because of testnet finalization... but it seems that there have been plenty of such failures in the past couple of days + the local sim passes locally.

This is only sort of true.

Looking at https://github.com/status-im/nim-beacon-chain/commits/devel since Jul 10 inclusive, there were:
14 commits, 10 of which going through CI, merged on Jul 10
3 commits. 2 going through CI on devel, on Jul 11
3 commits (but only two going through devel CI) on Jul 12
8 commits on Jul 13
2 commits on Jul 14

Of those, looking at specifically Jenkins failures (not the odd Travis and/or Azure failures, which have other causes):

- Of the 10 Jul 10 commits which went through CI, one (https://github.com/status-im/nim-beacon-chain/commit/87928e006962eae88c7e33ca73e32c9368ce6c03 which is  documentation fix) failed.

- Of the 2 devel CI Jul 11 commits, both succeeded in Jenkins and otherwise

- Likewise, of the 2 devel CI Jul 12 commits, both succeeded in Jenkins and otherwise

- Of the 8 Jul 13 commits, 2 failed, https://github.com/status-im/nim-beacon-chain/pull/1309 and https://github.com/status-im/nim-beacon-chain/pull/1289 where actually the first of those is somewhat concerning, because...

- The Jul 14 commits, both seemingly related to https://github.com/status-im/nim-beacon-chain/pull/1315, also failed in Jenkins.

So over the last 5 days (inclusive) we've gone from a 10% failure rate to a 25% failure rate to ... whatever it's going to be today.

I'm actually quite concerned about this. We've gotten stuck in weeks-long periods of bad finalization recently. I closed https://github.com/status-im/nim-beacon-chain/issues/1212 because things were looking good, but hey, a bunch of churn and now they may be on the way to broken again.

**Comment by onqtam** [comment]
pushed the changes but the use of VCs for local sims is still disabled by default - until we have normal stable finalization with plain old beacon nodes.

**Comment by tersec** [comment]
> pushed the changes but the use of VCs for local sims is still disabled by default - until we have normal stable finalization with plain old beacon nodes.

Sounds good.

**Pull request closed without merging**